ecommerce/api.py

```diff
@@ -480,3 +482,28 @@ def calculate_run_price(course_run, user):
     coupon = coupons[0]
     price = calculate_coupon_price(coupon, price, course_run.edx_course_key)
     return price, coupon
+
+
+def validate_prices():
+    """
+    Validate prices and financial aid discounts
+    Returns:
+        (list): List of validation errors
```

Comment:
You don't need parentheses around the return value type: `list: List of validation errors`